content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -93,7 +93,7 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
 1. Test to ensure the version you installed is up-to-date:
 
    ```bash
-   kubectl version --client
+   kubectl version --short
```

Comment:
```suggestion
   kubectl version --short --client
```

as otherwise this change tries to connect to a cluster.

Reply:
[kubernetes/kubectl#1216](https://github.com/kubernetes/kubectl/issues/1216) As far as I understood, doesn't this lead to the conclusion that "kubectl version --client" would work best for first timer users not having any cluster as --short has been depreciated and will be removed in the future?

Reply:
Unfortunately, `kubectl version --client` prints a different warning. As https://github.com/kubernetes/kubectl/issues/1216 explains, all the options for finding out what version of `kubectl` you're running print some warning. This is not ideal.

Reply:
The least bad option is, I think, to leave the page as it is now.